### ISSUE TITLE
rpn_bbox_loss and mrcnn_bbox_loss and mrcnn_mask_loss very high 

### ISSUE THREAD

**Issue by CodeXiaoLingYun** [matterport/Mask_RCNN#1650]
three loss : rpn_bbox_loss and mrcnn_bbox_loss and mrcnn_mask_loss 
i train the model for my dataset, my train.py was modified from shapes,and i redefined 

RPN_ANCHOR_SCALES = (2 * 6, 4 * 6, 8 * 6, 16 * 6, 32 * 6),
RPN_ANCHOR_RATIOS = [0.8, 1, 1.2].
NUM_CLASSES = 1 + 1  # background + pn
IMAGE_MIN_DIM = 256
IMAGE_MAX_DIM = 256

but the three losses are always too high ,not to go down, i do not know if the question is config or training data?
loss: 0.9434 - rpn_class_loss: 0.0117 - rpn_bbox_loss: 0.2421 - mrcnn_class_loss: 0.0663 - mrcnn_bbox_loss: 0.2227 - mrcnn_mask_loss: 0.4006
i know this question isn't very well described, but i want a  general way to modify it
data example:
![00000_1_01](https://user-images.githubusercontent.com/20829946/61588773-79b70f00-abd3-11e9-9054-e76c31c004b8.jpg)
![00000_1_01](https://user-images.githubusercontent.com/20829946/61588774-820f4a00-abd3-11e9-80d4-1ca2bfa9dde8.png)

**Comment by cpoptic** [comment]
Could you post your config file, including:

config.LEARNING_RATE, 
config.BACKBONE_STRIDES,
config.RPN_ANCHOR_SCALES,
config.RPN_ANCHOR_RATIOS,
config.RPN_ANCHOR_STRIDE

That would help diagnose your issue.

**Comment by ChienWong** [comment]
I have the same problem. Now ,rpn_bbox_loss: is 0.238, Although the loss continue to decrease, do quite slowly,1k image(about 10k samples) decrease 0.001.
What should i do ?

**Comment by Amrimn** [comment]
I have same probleme my rpn_bbox_loss and  mrcnn_bbox_loss are high do you have a idea what to do ?

thanks

**Comment by Altimis** [comment]
Hi. 
I suggest to change mrcnn_class_loss, mrcnn_bbox_loss and mrcnn_mask_loss in the LOSS_WEIGHTS  dictionary in your config file from 1 to 10 (you can force the model to learn by penalizing the loss function) 

```
LOSS_WEIGHTS = {
        "rpn_class_loss": 1.,
        "rpn_bbox_loss": 1.,
        "mrcnn_class_loss": 10.,
        "mrcnn_bbox_loss": 10., 
        "mrcnn_mask_loss": 10.}
```
